docs/src/reference.md

```diff
@@ -25,7 +25,7 @@ Pages = ["src/solver.jl"]
 
 ```@autodocs
 Modules = [COBREXA]
-Pages = ["src/builders/core.jl"]
+Pages = ["src/builders/core.jl", "src/builders/mmdf.jl"]
```

Comment:
there is no core anymore, pls rebase atop the master (this belongs to reference/builders.md)